[2022-05-05 13:32:01,164] {taskinstance.py:1043} INFO - Dependencies all met for <TaskInstance: big_finance_dag.fetch_data_from_big_finance manual__2022-05-05T11:31:58.696339+00:00 [queued]>
[2022-05-05 13:32:01,168] {taskinstance.py:1043} INFO - Dependencies all met for <TaskInstance: big_finance_dag.fetch_data_from_big_finance manual__2022-05-05T11:31:58.696339+00:00 [queued]>
[2022-05-05 13:32:01,168] {taskinstance.py:1249} INFO - 
--------------------------------------------------------------------------------
[2022-05-05 13:32:01,168] {taskinstance.py:1250} INFO - Starting attempt 1 of 2
[2022-05-05 13:32:01,168] {taskinstance.py:1251} INFO - 
--------------------------------------------------------------------------------
[2022-05-05 13:32:01,181] {taskinstance.py:1270} INFO - Executing <Task(PythonOperator): fetch_data_from_big_finance> on 2022-05-05 11:31:58.696339+00:00
[2022-05-05 13:32:01,183] {standard_task_runner.py:52} INFO - Started process 25918 to run task
[2022-05-05 13:32:01,185] {standard_task_runner.py:79} INFO - Running: ['airflow', 'tasks', 'run', 'big_finance_dag', 'fetch_data_from_big_finance', 'manual__2022-05-05T11:31:58.696339+00:00', '--job-id', '11', '--raw', '--subdir', 'DAGS_FOLDER/big_finance_dag.py', '--cfg-path', '/tmp/tmpsaoz2e96', '--error-file', '/tmp/tmpwqdsmebp']
[2022-05-05 13:32:01,186] {standard_task_runner.py:80} INFO - Job 11: Subtask fetch_data_from_big_finance
[2022-05-05 13:32:01,221] {logging_mixin.py:109} INFO - Running <TaskInstance: big_finance_dag.fetch_data_from_big_finance manual__2022-05-05T11:31:58.696339+00:00 [running]> on host DESKTOP-O4M0F04.localdomain
[2022-05-05 13:32:01,263] {taskinstance.py:1446} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_EMAIL=airflow@example.com
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=big_finance_dag
AIRFLOW_CTX_TASK_ID=fetch_data_from_big_finance
AIRFLOW_CTX_EXECUTION_DATE=2022-05-05T11:31:58.696339+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-05-05T11:31:58.696339+00:00
[2022-05-05 13:32:01,264] {logging_mixin.py:109} INFO - We are getting data from big finance api issou
[2022-05-05 13:32:01,264] {python.py:175} INFO - Done. Returned value was: None
[2022-05-05 13:32:01,270] {taskinstance.py:1278} INFO - Marking task as SUCCESS. dag_id=big_finance_dag, task_id=fetch_data_from_big_finance, execution_date=20220505T113158, start_date=20220505T113201, end_date=20220505T113201
[2022-05-05 13:32:01,318] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-05-05 13:32:01,327] {local_task_job.py:264} INFO - 0 downstream tasks scheduled from follow-on schedule check
